programs/server/play.html

```diff
@@ -1289,28 +1290,103 @@
     query_area.value = window.atob(window.location.hash.substr(1));
 }
 
+function parseQueries(str)
```

Comment:
This does not work because it isn't identical to the ClickHouse SQL lexer.

Examples:
```
SELECT `;`, ";", 'hello''world;', $;$;$;$, ‘he;llo’;
INSERT INTO table FORMAT RowBinary
;
```